frontend/src/Components/Floaters/Floater.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "../../Style/Event.css";
+import "../../Style/Events.css";
+
+const Floater = props => {
+    return (
+        <span>
+            <h1>Floaters</h1>
+            <p>First Name: {props.floater_fname}</p>
```

Comment:
try to use `props.name` and you'll see the difference. See my comment in `Floaters.js`